### PR TITLE
[unitaryhack] Update GUI workflow for the current version of KLayout

### LINKED ISSUE #11: Point-and-click workflow tutorial outdated
With new KLayout and KQCircuits versions some things have changed so that parts of ["point-and-click" workflow tutorial](https://iqm-finland.github.io/KQCircuits/user_guide/gui_workflow.html) don't work as described or are confusing. At least:

- Part 4.4-5.4: Not easy to select qubit because waveguide cells are selected instead.
	* Alternative way to select a qubit is to drag a selection box around it. Pressing "q" can then be used to open the "object properties" window.
- Part 6.5: The button is now a magnifying glass instead of "...".
	* Can mention both, so that both older and new KLayout versions can be used
- Part 7.1: Ground grid is not visible by default, should tell to unhide it in the layers list.

The end result should be such that everything works when following exactly the instructions in the tutorial.

### PR DESCRIPTION
**Pull request by Renaud2002**
Hi, this resolves #11 as part of the unitaryhack issues. I have added the update suggested in the issue and the ones I found while going through the program.

### CODE REVIEW

File: docs/user_guide/gui_workflow.rst
Comment (jkotilahti) [inline docs/user_guide/gui_workflow.rst:65]: Looks like in this part you changed it so that it changes a parameter of the Demo chip (`Include coupler between qubits`) instead of `Coupler lengths` parameter of a single qubit. It shouldn't be possible to change the demo chip parameters at this point if it was turned to static as explained in earlier step of the tutorial. Probably the confusion was because you didn't turn Demo chip to static and because the selection of the qubit does not work easily by double clicking. I suggest changing this part of the tutorial to the following:

```
#. **Disable a coupling port for the qubit.**

   #. From the *Tools* panel select *Select*.
   #. Drag a selection box around the qubit.
   #. Click ``Shift+F2``
   #. Press ``q``.
   #. In the new *Instance Properties* window locate a tab *PCell
      parameters*.
   #. Scroll down in the list of parameters to locate *Coupler lengths*.
   #. Change the value in the text box to ``100,160,0``.
   #. Click ``Ok``.
```

Please check if this makes sense to you and change accordingly.

File: docs/user_guide/gui_workflow.rst
Comment (jkotilahti) [inline docs/user_guide/gui_workflow.rst:72]: This line should be removed and the previous line should be changed to say `toolbar` instead of `*Tools* panel`.

File: docs/user_guide/gui_workflow.rst
Comment (jkotilahti) [inline docs/user_guide/gui_workflow.rst:73]:
```suggestion
   #. In the new *Editor Options* window locate *Library*.
```

This will be the name of the window if you click "Instance" from toolbar (instead of double-clicking existing qubit).

File: docs/user_guide/gui_workflow.rst
Comment (jkotilahti) [inline docs/user_guide/gui_workflow.rst:75]:
```suggestion
   #. Click on a button ``🔍`` a little bit to the left from the *Library* 
```

**Review by jkotilahti — CHANGES_REQUESTED** [review]
@Renaud2002 Thanks, mostly the changes look good! But there are some changes that should be done we can accept this PR, please see my comments above.

File: docs/user_guide/gui_workflow.rst
Comment (Renaud2002) [reply docs/user_guide/gui_workflow.rst:72]: @jkotilahti Hi, I am not so much which line you are saying should be removed do you mean the ```#. Double-click on the qubit```

Comment (jkotilahti) [reply docs/user_guide/gui_workflow.rst:72]: Yes, removing the `#. Double-click on the qubit` line is what I mean.

**Comment by jkotilahti** [comment]
Thanks @Renaud2002, I think everything looks good now! You can find the CLA to sign here https://meetiqm.com/developers/clas/individual/ (it can also be found by following links in https://github.com/iqm-finland/KQCircuits#contributing). Note that it might take until next week before we can merge this if you didn't sign the CLA yet.

**Comment by Renaud2002** [comment]
Hi @jkotilahti I just sign the QM CLA please let me know if you see it, thank you very much for all your help.

**Comment by Renaud2002** [comment]
@jkotilahti Thank you very much for all your help through the process I am grateful.

**Review by jkotilahti — APPROVED** [review]
@Renaud2002 I received confirmation about the CLA and will merge this now. Thank you for contributing to KQCircuits!

**Pull request merged**
